app/services/application_purger.rb

```diff
@@ -27,22 +26,6 @@ def delete_orphan_stored_documents
     end
   end
 
```

Comment:
One benefit of moving all dynamic scoping to Income, Outgoings, and Capital models is that the CrimeApplication dependent destroy function works as expected and this code can go :)